docker/environments/dart-2.10.4/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM google/dart:2.10.4
+
+LABEL maintainer="team@appwrite.io"
+
+RUN apk add tar
+
+RUN mkdir /usr/local/src
```

Comment:
I think we need to set PUB_CACHE env var here by default like in other langs that use a central dependencies dir